Header header
geometry_msgs/Pose[] poses
int32 nrDetectedLShapes
geometry_msgs/Point[] idx
int32[] LShapesIdxs
geometry_msgs/Point[] LShapesPos
geometry_msgs/Quaternion[] LShapesOri
